providers/openai/encode.go

```
@@ -423,20 +458,36 @@ func encodeAssistantCodeInterpreterCallContent(c *CodeInterpreterCallContent) (r
+	return encodeUserMessageWithPromptCacheBreakpoint(message, false)
+}
+
+func encodeUserMessageWithPromptCacheBreakpoint(message *llm.Message, breakpoint bool) ([]responses.ResponseInputItemUnionParam, error) {
 	if message.Role != llm.User {
 		return nil, fmt.Errorf("message role is not user")
 	}
-	return encodeInputMessage(message)
+	return encodeInputMessageWithPromptCacheBreakpoint(message, breakpoint)
 }
 
 func encodeInputMessage(message *llm.Message) ([]responses.ResponseInputItemUnionParam, error) {
```

golangci-lint (2.12.2)

[error] func encodeInputMessage is unused

(unused)